src/main/java/com/liferay/ide/idea/ui/modules/LiferayWorkspaceBuilder.java

```diff
@@ -117,9 +149,23 @@ protected void initWorkspace(Project project) {
 		component.setValue(selectedLiferayVersionProperty, _liferayVersion);
 
 		BladeCLI.execute(sb.toString());
+
+		if ("LiferayGradleWorkspce".equals(_liferayProjectType)) {
```

Comment:
Is this the correct constant?  it is missspelled. Should be  "LiferayGradleWorkspace"  Probably should be a constant?